scripts/test_updates_pg12.sh

```diff
@@ -6,11 +6,20 @@ set -o pipefail
 SCRIPT_DIR=$(dirname $0)
 echo $SCRIPT_DIR
 
-TAGS="1.7.0-pg12 1.7.1-pg12 1.7.2-pg12 1.7.3-pg12 1.7.4-pg12 1.7.5-pg12 2.0.0-rc1-pg12 2.0.0-rc2-pg12 2.0.0-rc3-pg12 2.0.0-rc4-pg12 2.0.0-pg12 2.0.1-pg12 2.0.2-pg12 2.1.0-pg12"
+TAGS="1.7.0-pg12 1.7.1-pg12 1.7.2-pg12 1.7.3-pg12 1.7.4-pg12 1.7.5-pg12"
```

Comment:
Since we only have a single version supporting pg13 there is nothing to add yet. Also, this task is about refactoring existing tests, not adding new ones, and I prefer to keep those separate to not conflate issues necessarily.